app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,69 @@
+class PostsController < ApplicationController
+  before_action :set_post, only: %i[show edit update destroy]
+  before_action :authenticate_user!, except: %i[index]
+
+  # GET /posts or /posts.json
+  def index
+    @posts = Post.all
+  end
+
+  # GET /posts/1 or /posts/1.json
+  def show; end
+
+  # GET /posts/new
+  def new
+    # @post = current_user.posts.build
```

Comment:
Remove the commented out code. It makes it difficult to understand the codebase.